### PR TITLE
Add info about changing bucket sharing type

### PR DESCRIPTION
**Pull request by pivnicek**

### CODE REVIEW

File: storage/buckets/sharing/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ any tables you create in the shared bucket become immediately available in the d
 You can specify who can link the bucket to a project:
 
 - **Organization Member** --- Only organization members are able to link the shared bucket to a project.
-- **Project Member** --- Every project member is able to link the shared bucket to a project.
+- **Project Member** --- Every project member in the organisation is able to link the shared bucket to a project.
```
Comment (odinuv) [inline storage/buckets/sharing/index.md]: so is it the member of project or the member of organization? if the latter, how is it different from the first sharing option?

Comment (odinuv) [reply storage/buckets/sharing/index.md]: exactly, then i don't get why i now reads:

> - **Project Member** --- Every project member in the organisation is able to link the shared bucket to a project.

Comment (pivnicek) [reply storage/buckets/sharing/index.md]: I just wanted to emphasize that project members from *other* organizations can not access it.

Comment (odinuv) [reply storage/buckets/sharing/index.md]: ah!

Comment (pivnicek) [reply storage/buckets/sharing/index.md]: So maybe?

- Project Member --- Any project member under this organisation can link the shared bucket to their project

Comment (odinuv) [reply storage/buckets/sharing/index.md]: hm, i've come up with this:

- **Project Member** --- To be able to link the bucket to a project, a person must be a user of the destination project (that project must be in the same organization as the source project).
- **Organization Member** --- To be able to link the bucket to a project, a person must be a user of the destination project 
(that project must be in the same organization as the source project) *and also a member of that organization*.

but not really happy with it, perhaps @hhanova will help? :)

Comment (pivnicek) [reply storage/buckets/sharing/index.md]: I think that's an improvement!  if not a little verbose :).  It's really amazing how difficult it is

**Comment by odinuv** [comment]
@pivnicek how about now?

File: storage/buckets/sharing/index.md [hunk 2]
```diff
@@ -7,22 +7,26 @@ permalink: /storage/buckets/sharing/
...
 any tables you create in the shared bucket become immediately available in the destination projects.
 
 ## Sharing Type
 You can specify who can link the bucket to a project:
 
-- **Organization Member** --- Only organization members are able to link the shared bucket to a project.
-- **Project Member** --- Every project member in the organisation is able to link the shared bucket to a project.
+- **Project Member** --- To be able to link the bucket to a project, a person must be a user of the destination project.
+- **Organization Member** --- To be able to link the bucket to a project, a person must be a user of the destination project
+*and also a member of that organization*.
```
Comment (pivnicek) [inline storage/buckets/sharing/index.md]: This wording is still confusing to me.  To me from a language POV, all project members would be members of the organisation.  I wish we could change it to role instead.  Something like `*and also be an administrator of that organization *`

Comment (odinuv) [reply storage/buckets/sharing/index.md]: understand, but i'd like to honor the UI naming
image

which unfortunately is also chaos https://github.com/keboola/kbc-ui/issues/2127

Comment (pivnicek) [reply storage/buckets/sharing/index.md]: I see, ok I guess this is fine and we can update it when the UI changes

**Pull request merged**